editor/assets/default_file_content/ts

```diff
@@ -1,42 +1,49 @@
-
-import { _decorator, Component, Node } from 'cc';
+import { _decorator, Component } from 'cc';
 const { ccclass, property } = _decorator;
 
-/**
- * Predefined variables
- * Name = <%Name%>
- * DateTime = <%DateTime%>
- * Author = <%Author%>
- * FileBasename = <%FileBasename%>
- * FileBasenameNoExtension = <%FileBasenameNoExtension%>
- * URL = <%URL%>
- * ManualUrl = <%ManualUrl%>
- *
- */
- 
 @ccclass('<%Name%>')
 export class <%Name%> extends Component {
-    // [1]
-    // dummy = '';
+    start() {}
 
-    // [2]
-    // @property
-    // serializableDummy = 0;
+    update(deltaTime: number) { }
+}
+
+/**
+ * GENERATE IGNORE
+ * Use the "GENERATE IGNORE" flag in comments to not generate these commented code in newly created script files.
+ * 
+ * Predefined Variables
+ * You can use predefined variables below, they can help you such as decide whether to use 'Camel Format Class Name'.
```

Comment:
You can use predefined variables below to setup your scripting preference. For example, whether to use camel case style.

Reply:
ok